### PR TITLE
Add some operations to ndarray

### PR DESCRIPTION
**Pull request by kbrose**
I just went through the `ndarray` docs alphabetically and added some operations. I wanted to submit for review before I got too far -- is this an approach you would consider including, or are you looking to get all the operations typed in some other way (e.g. auto-generating per #15)?

### CODE REVIEW

**Comment by kbrose** [comment]
Is this repo still being developed?

File: numpy-stubs/__init__.pyi [hunk 1]
```diff
@@ -368,6 +368,58 @@ class ndarray(_ArrayOrScalarCommon, Iterable, Sized, Container):
     def flatten(self, order: str = ...) -> ndarray: ...
     def ravel(self, order: str = ...) -> ndarray: ...
     def squeeze(self, axis: Union[int, Tuple[int, ...]] = ...) -> ndarray: ...
+    # Operations
+    @overload
+    def all(
+        self, axis: None = ..., out: None = ..., keepdims: Optional[bool] = ...
+    ) -> bool_: ...
+    @overload
+    def all(
```
Comment (person142) [inline numpy-stubs/__init__.pyi]: Note that specifying an axis doesn't necessarily mean you will get an array out; e.g.

```python
>>> x = np.array([True, True])
>>> x.all(axis=0)
True
```

And the analogue with more axes:

```python
>>> x = np.ones((2, 2), dtype=np.bool_)
>>> x.all(axis=(0, 1))
True
```

If we had `Literal` we could make that guarantee for when `keepdims = True`, but we can't do that yet.

File: numpy-stubs/__init__.pyi [hunk 2]
```diff
@@ -368,6 +368,58 @@ class ndarray(_ArrayOrScalarCommon, Iterable, Sized, Container):
     def flatten(self, order: str = ...) -> ndarray: ...
     def ravel(self, order: str = ...) -> ndarray: ...
     def squeeze(self, axis: Union[int, Tuple[int, ...]] = ...) -> ndarray: ...
+    # Operations
+    @overload
+    def all(
```
Comment (person142) [inline numpy-stubs/__init__.pyi]: You can specify a 0d array as the `out` even when not specifying an axis, e.g.

```python
>>> x = np.array([True, True])
>>> out = np.array(True, dtype=np.bool_)
>>> x.all(out=out)
array(True)
>>> out
array(True)
```

File: numpy-stubs/__init__.pyi [hunk 3]
```diff
@@ -368,6 +368,58 @@ class ndarray(_ArrayOrScalarCommon, Iterable, Sized, Container):
...
+        self, axis: None = ..., out: None = ..., keepdims: Optional[bool] = ...
+    ) -> bool_: ...
+    @overload
+    def all(
+        self, axis: int, out: Optional[ndarray] = ..., keepdims: Optional[bool] = ...
+    ) -> ndarray: ...
+    @overload
+    def any(
```
Comment (person142) [inline numpy-stubs/__init__.pyi]: Similar comments here as with `all`.

File: numpy-stubs/__init__.pyi [hunk 4]
```diff
@@ -368,6 +368,58 @@ class ndarray(_ArrayOrScalarCommon, Iterable, Sized, Container):
...
+        self, axis: None = ..., out: None = ..., keepdims: Optional[bool] = ...
+    ) -> bool_: ...
+    @overload
+    def any(
+        self, axis: int, out: Optional[ndarray] = ..., keepdims: Optional[bool] = ...
+    ) -> ndarray: ...
+    @overload
+    def argmax(self, axis: None = ..., out: Optional[ndarray] = ...) -> integer: ...
+    @overload
+    def argmax(self, axis: int, out: Optional[ndarray] = ...) -> ndarray: ...
```
Comment (person142) [inline numpy-stubs/__init__.pyi]: Similarly we might get a scalar output here:

```python
>>> x = np.arange(4)
>>> x.argmax(axis=0)
3
```

**Comment by kbrose** [comment]
I'm no longer motivated to finish this at this point in time.

**Pull request closed without merging**